lib/polyfill/v2_6/matrix.rb

```diff
@@ -0,0 +1,15 @@
+require 'matrix'
+
+module Polyfill
+  module V2_6
+    module Matrix
+      def antisymmetric?
+        Matrix.Raise ErrDimensionMismatch unless square?
```

Comment:
@AaronLasseigne How should this new error constant be brought in by Polyfill?

Reply:
In the past I've avoided bringing in anything that can't be refined into Ruby. This includes new errors. I handled this for `#delete_suffix!` (Ruby 2.5) by using the parent of the error, marking the method as partially implemented, and noting the difference.